### PR TITLE
[RM Ch03.6] Model realization

### PR DESCRIPTION
**Pull request by walterkozlowski**
Added a new section (3.6) on sample model realization, illustrated in a diagram.

### CODE REVIEW

File: doc/ref_model/chapters/chapter03.md [hunk 1]
```diff
@@ -202,3 +202,12 @@ Principles that should be followed during the development and definition of the
...
+## 3.6 Sample reference model realization
+The following diagram presents an example of the realization of the reference model, where a virtual infrastructure layer contains three coexisting but different types of implementation: a typical IaaS using VMs and a hypervisor for virtualisation, a CaaS on VM/hypervisor, and a CaaS on bare metal. This diagram is presented for illustration purposes only and it does not precluded validity of many different combinations of implementation types. Note that the model enables several potentially different SDN controllers orchestrating different level of resources (virtual and/or hardware). Management clients can manage virtual resources via Virtual Infrastructure Manager (Container Infrastructure Service Manager for CaaS, or Virtual Infrastructure Manager for IaaS), or alternatively hardware infrastructure resources via hardware infrastructure manager.  The latter situation may occur for instance when an orchestrator (an example of a management client) is involved in provisioning the physical network resources with the assistance of SDN controllers. Also, this realization example would enable implementation of a programmable fabric.
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter03.md:208]: Can we deleted SDN and then controllers will be valid for any type of resource. Typo suggestions
```suggestion
The following diagram presents an example of the realization of the reference model, where a virtual infrastructure layer contains three coexisting but different types of implementation: a typical IaaS using VMs and a hypervisor for virtualisation, a CaaS on VM/hypervisor, and a CaaS on bare metal. This diagram is presented for illustration purposes only and it does not preclude validity of many other different combinations of implementation types. Note that the model enables several potentially different controllers orchestrating different type of resources (virtual and/or hardware). Management clients can manage virtual resources via Virtual Infrastructure Manager (Container Infrastructure Service Manager for CaaS, or Virtual Infrastructure Manager for IaaS), or alternatively hardware infrastructure resources via hardware infrastructure manager.  The latter situation may occur for instance when an orchestrator (an example of a management client) is involved in provisioning the physical network resources with the assistance of the controllers. Also, this realization example would enable implementation of a programmable fabric.
```

File: doc/ref_model/chapters/chapter03.md [hunk 2]
```diff
@@ -202,3 +202,12 @@ Principles that should be followed during the development and definition of the
...
+
+<p align="center"><img src="../figures/ch03-model-realization-diagram.png" alt="Reference model realization example" Title="Reference model realization example" width="65%"/></p>
+<p align="center"><b>Figure 3-6:</b> NFVI Physical Resources</p>
+
+The terms: Container Infrastructure Service Instance and Container Infrastructure Service Manager should be understand as defined in ETSI GR NFV-IFA 029 https://www.etsi.org/deliver/etsi_gr/NFV-IFA/001_099/029/03.03.01_60/gr_NFV-IFA029v030301p.pdf. More detailed deployment examples cna be found in Section 4.3 of this Reference Model document.
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter03.md:213]:
```suggestion
The terms: Container Infrastructure Service Instance and Container Infrastructure Service Manager should be understood as defined in [ETSI GR NFV-IFA 029](https://www.etsi.org/deliver/etsi_gr/NFV-IFA/001_099/029/03.03.01_60/gr_NFV-IFA029v030301p.pdf). More detailed deployment examples can be found in [Section 4.3](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter04.md#43-networking) of this Reference Model document.
```

**Comment by rabi-abdel** [comment]
@walterkozlowski can you please address comments above?

**Comment by walterkozlowski** [comment]
@rabi-abdel, I have accepted @pgoyal01 suggestions.

**Comment by rabi-abdel** [comment]
@pgoyal01 can you please re-review/approve?

**Comment by walterkozlowski** [comment]
@TFredberg please approve this PR and we will be able to merge

**Comment by walterkozlowski** [comment]
@ulikleber , can you approve if you are OK with this (we talked few times before) so we can merge into master.  It does not stop us from future work and improvements.

**Pull request merged**